public/css/header.module.css

```diff
@@ -1,47 +1,60 @@
 .header {
-    color: #fff;
-    background-color: rgba(255, 105, 105, 0.6);
-    padding: 0;
-    margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: #fff;
+  background-color: rgba(255, 105, 105, 0.6);
+  padding: 0;
+  padding-block: 10px;
+  width: 100%;
+  position: absolute;
+  user-select: none;
+
+  & img {
+    width: 50px;
+    aspect-ratio: 1;
+    filter: invert(100%);
+  }
+
+  & > * {
     display: flex;
-    justify-content: space-between;
     align-items: center;
-    width: 100%;
-    position: absolute;
-    user-select: none;
-
-    & img {
-        width: 50px;
-        aspect-ratio: 1;
-        filter: brightness(0) saturate(100%) invert(99%) sepia(60%) saturate(0%) hue-rotate(261deg) brightness(116%) contrast(100%);
-    }
-    &>* {
-        display: flex;
-        align-items: center;
-        gap: 10px;
-        padding-inline: 10px;
-    }
-    
-    & h1 {
-        font-size: 42px;
-    }
-}
+    gap: 10px;
+    padding-inline: 10px;
+  }
+
+  & * {
+    margin: 0;
+  }
 
+  & h1 {
+    margin: 0;
+    font-size: 42px;
+  }
+  & > div:first-child > div:first-child {
+    display: flex;
+    gap: 10px;
+  }
+}
 
 @media only screen and (max-width: 1200px) {
-    .header {
-        background-color: rgba(255, 255, 255, 0);
-        justify-content: center;
-        padding: 0;
-        top: 10%;
-        z-index: 1;
-
-        &>h1 {
-            display: none;
-        }
-
-        & div {
-            flex-direction: row-reverse;
-        }
+  .header {
+    background-color: rgba(255, 255, 255, 0);
+    justify-content: center;
+    padding: 0;
+    top: 10%;
+    z-index: 1;
+
+    & > div {
+      flex-direction: column;
     }
-}
\ No newline at end of file
+
+    & > div:last-child h1 {
+      display: none;
+    }
```

Comment:
literally showing you that there is no use for that div😭😭😭

Reply:
its in the media query, its for hiding the date.

Reply:
the date is directly in the header. why a `div:last-child`?